applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc

```diff
@@ -7,7 +7,8 @@ toc::[]
 
 The *Developer* perspective in the web console provides you the following options from the *Add* view to create applications and associated services and deploy them on {product-title}:
 
-image::odc_add_view.png[Add View]
+.Add View
```

Comment:
All titles added throughout this PR should be sentence case.

Reply:
@bergerhoffer - I have made the changes suggested by you, request you to please review and let me know if you are ok with the changes. Thanks